server/src/server.ts

```diff
@@ -2,5 +2,5 @@ import app from './app';
 const port = 3000;
```

Comment:
We should use a different port like 4000 since we will be using 3000 for the client.